Jenkinsfile

```diff
@@ -21,6 +21,8 @@ node('vetsgov-general-purpose') {
   // setupStage
   dockerContainer = commonStages.setup()
 
+  commonStages.build(ref, dockerContainer, 'local', 'vagovprod', false, false)
```

Comment:
Was this intentional? Seemed like only the integration pipeline needed the build to be added.

Reply:
yeah, forgot to pull it back to a draft

Reply:
@U-DON ready now